### ISSUE TITLE
Some questions about English version

### ISSUE THREAD

**Issue by goto-vlad** [dennis714/RE-for-beginners#139]
There are some words in the text that sound slightly off, but i am not sure whether they should be changed or not, because some of them are quite fine as it is and it may very well be false alarm on my part in some cases, so i apologize in advance.

Proposed changes:

1. > I didn't try this by myself

   > I didn't try this myself

[some explanation on english.stackexchange](http://english.stackexchange.com/questions/100221/i-did-it-by-myself-vs-i-did-it-myself)

2. > I didn't able to

   > I wasn't able to

3. > point not belongs to Mandelbrot set

   > point doesn't belong to Mandelbrot set

4. > any character other than digit will result unsigned number

   > any character other than digit will result **in** unsigned number

Completely unrelated, but some words "aesthetical[ly]" in the book contain one symbol "ae" instead of two different letters.Source files seem to show that this was intentional (it's written there as "\ae{}").
To add to the confusion, two of the words "aesthetical[ly]" written this way, and two - with different letters.
I am nowhere near fluent in LaTex, but this seems a little strange to me.

**Comment by dennis714** [comment]
Thanks a lot. https://github.com/dennis714/RE-for-beginners/commit/06161a5b29bccffa72afa7e98839c45ab5d40686

\ae{} is ligature, it renders to a single character with A and E fused together.
It's not a necessity, just to make it a little fancy.

Not sure about difference between aesthetical and aesthetically. Which is better?

**Comment by goto-vlad** [comment]
I think the use of "by myself" in the "Call for translators" is correct, because its purpose is to emphasize the fact that you are the sole author of the English and Russian versions of the book.

#141

**Comment by goto-vlad** [comment]
What i meant to say was that usage of "ae" character is inconsistent.
Compare, for example, these two excerpts from the book:

![test](https://cloud.githubusercontent.com/assets/24530738/21221594/a3d00772-c2c6-11e6-944e-e2277ae28cf6.png)

**Comment by dennis714** [comment]
OK: https://github.com/dennis714/RE-for-beginners/commit/5a3099de3e5f6c81d64d2b3321f73b4b6005424f

**Comment by goto-vlad** [comment]
[link](https://github.com/dennis714/RE-for-beginners/blob/521bf8cdd856dda20247dba41f9de4a9a72ccb61/patterns/MIPS/big_constants_EN.tex#L11)

I think that the word "size" or "length" is missing here.

**Comment by dennis714** [comment]
Yes, you're right: https://github.com/dennis714/RE-for-beginners/commit/767dfa1fb41b595dd4b7c7ede9bee2170ea08e9f

How to credit you properly? Your name?

**Comment by goto-vlad** [comment]
> How to credit you properly?

I think you already did.

[thanks_EN.tex#L41](https://github.com/dennis714/RE-for-beginners/blob/767dfa1fb41b595dd4b7c7ede9bee2170ea08e9f/thanks_EN.tex#L41)

I am just really enjoying reading you book, i would even go as far as to say that it is the best book about assembly language. Despite its a little "scary" name, it helped me immensely with understanding, for example, the code generated by the compiler (before there was too much noise for me), but now i am starting to observe certain patterns, albeit simple ones. And i registered on Github because of this book, which is also a plus.
